ERCS/erc-xxxx.md

```diff
@@ -0,0 +1,499 @@
+---
+eip: <to be assigned>
+title: Store, Table-Based Introspectable Storage
```

Comment:
```suggestion
title: Table-Based Introspectable Storage
```
Maybe simplify the title

Reply:
We would like to keep the name for now. Thanks for the feedback!

Reply:
For context - we feel it's important to keep the term "Store" in the title since it's the name of the pattern, ie all events are prefixed with `Store`